generator/generator.yml

```diff
@@ -101,3 +101,41 @@ modules:
       - 1.3.6.1.4.1.1718.3.2.2  # infeedTable
       - 1.3.6.1.4.1.1718.3.2.3  # outletTable
 
+# A10 Ax Load Balancers
+#
+# The MIB files are available for download through the load balancer GUI:
```

Comment:
It has to be a strict requirement, otherwise only people who own every piece of hardware in this config will be able to contribute changes.